docs/catalog/cloud/Design for independent scaling of logical components.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Design for independent scaling of logical components
+
+## Description
+A microservice architecture may reduce the compute resources required, as it allows for independent scaling of its logical components and ensures they are scaled according to the demand.
```

Comment:
"scaled according to the demand" of each individual component. The independent scaling is the value here so you should be explicit in calling it out.